lib/rails/generators/anycable/setup/docker/docker_generator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module AnyCableRailsGenerators
+  module Setup
+    # Generator to help set up AnyCable in Docker environment
+    class DockerGenerator < ::Rails::Generators::Base
+      namespace "anycable:setup:docker"
+
+      def info
+        say "Docker configs are not standardized."
+        say "So, right now, we cannot continue installation process."
+        say "🛈 But, you can find an example of workable configs here: "
+        say "  👉 https://github.com/evilmartians/evil_chat/tree/feature/anycable"
```

Comment:
Let's add a `docker-compose.yml` snippet instead for now:

```iml
anycable-ws:
  image: anycable/anycable-go:v0.6.4
  ports:
    - '3334:3334'
  environment:
    PORT: 3334
    REDIS_URL: redis://redis:6379/0
    ANYCABLE_RPC_HOST: anycable-rpc:50051
  depends_on:
    - anycable-rpc
    - redis

anycable-rpc:
  <<: *backend
  command: bundle exec anycable   
  ports:
    - '50051'
```

We will replace it with the link to the documentation later.